dotnet-tools.json

```diff
@@ -21,4 +21,4 @@
       ]
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
Revert.

Reply:
Should we change the editorconfig?

Reply:
Could do. `dotnet tool` removes the trailing newline, which is why it usually isn't there. Probably shouldn't be edited by hand.

Reply:
Interesting. IMO, it would be better to enforce in the config (same with the solution), for pathological people like me who hand edit these files sometimes.